packages/strapi-admin/admin/src/containers/App/index.js

```diff
@@ -49,7 +55,7 @@ export class App extends React.Component { // eslint-disable-line react/prefer-s
 }
 
 App.contextTypes = {
-  router: PropTypes.object.isRequired,
+  router: PropTypes.object,
```

Comment:
I'm not sure the router context is needed anymore. The router needs to be updated in another PR